modules/postgresql/outputs.tf

```diff
@@ -20,12 +20,12 @@ output "instance_name" {
   description = "The instance name for the master instance"
 }
 
-output "instance_address" {
+output "instance_ip_address" {
```

Comment:
This change is reasonable? @morgante

Reply:
I think we should leave it as is.

Reply:
OK.
We'll be able to deprecate this and rename in the future (probably next major version bump?)

Reply:
Fixed: https://github.com/terraform-google-modules/terraform-google-sql-db/pull/57/commits/865b976daab04259fa953ee9de04bd2d9e156c3a
Issue opened: #58